app/models/user.rb

```diff
@@ -3,7 +3,7 @@ class User < ApplicationRecord
   has_many :videos, through: :user_videos
 
   validates :email, uniqueness: true, presence: true
-  validates :password, presence: true
+  validates :password, presence: true, on: :update, allow_blank: true
```

Comment:
alows user to be updated without a password

Reply:
ah this answers my earlier question. Can maybe use update_attribute instead so we don't have to mess with password validation since it is related to security?